### PR TITLE
Install selected version of Sat server from CDN

### PR DESCRIPTION
**Pull request by sghai**
I just updated the `product_install` function and `repository` to enable subscriptions based on given sat version for CDN distribution.

Will update the jenkins installer job later.

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -833,7 +833,7 @@ def sam_upstream_install(admin_password=None):
 
 
 def product_install(distribution, create_vm=False, certificate_url=None,
-                    selinux_mode=None):
+                    selinux_mode=None, sat_version=None):
```
Comment (elyezer) [inline automation_tools/__init__.py]: s/sat_version/sat_cdn_version

File: automation_tools/repository.py [hunk 2]
```diff
@@ -125,7 +125,8 @@ def create_custom_repos(**kwargs):
         repo_file.close()
 
 
-def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True):
+def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True,
+                           version='6.1'):
```
Comment (elyezer) [inline automation_tools/repository.py]: s/version/cdn_version

**Comment by elyezer** [comment]
product_install should fail earlier (before doing anything) if it is installing from CDN and don't get an expected version ('6.0', '6.1'). Also the enable_satellite_repos task should raise an exception for the same reason and the same way before doing anything.

**Comment by elyezer** [comment]
ACK, pending comment.

**Comment by elyezer** [comment]
ACK

**Comment by Ichimonji10** [comment]
ACK

**Comment by sthirugn** [comment]
@sghai this also needs jenkins update correct?

**Comment by sghai** [comment]
Yes. I filed an issue https://github.com/SatelliteQE/robottelo-ci/issues/44

**Pull request merged**